runners/core-java/src/main/java/org/apache/beam/runners/core/ReduceFnRunner.java

```diff
@@ -69,25 +71,25 @@
  * Manages the execution of a {@link ReduceFn} after a {@link GroupByKeyOnly} has partitioned the
  * {@link PCollection} by key.
  *
- * <p>The {@link #onTrigger} relies on a {@link TriggerRunner} to manage the execution of
- * the triggering logic. The {@code ReduceFnRunner}s responsibilities are:
+ * <p>The {@link #onTrigger} relies on a {@link TriggerStateMachineRunner} to manage the execution
+ * of the triggering logic. The {@code ReduceFnRunner}s responsibilities are:
  *
  * <ul>
- * <li>Tracking the windows that are active (have buffered data) as elements arrive and
- * triggers are fired.
- * <li>Holding the watermark based on the timestamps of elements in a pane and releasing it
- * when the trigger fires.
+ * <li>Tracking the windows that are active (have buffered data) as elements arrive and triggers are
+ *     fired.
```

Comment:
I did have to re-wrap one line, and `google-java-format` always autoformats a full javadoc comment at a time.